src/Model/Term.php

```diff
@@ -147,64 +201,73 @@ public static function insertFromTagFieldByItemId($itemid, $tags)
 					$term = substr($tag, 1);
 				}
 			} else { // This shouldn't happen
```

Comment:
If this shouldn't happen, maybe a additional `Logger::notice()` would be helpful?